isso/views/comments.py

```diff
@@ -1562,6 +1590,77 @@ def latest(self, environ, request):
                 "Unavailable because 'latest-enabled' not set by site admin"
             )
 
+        return self._latest(environ, request, "1")
+
+
+    def check_auth(self, username, password):
+        admin_password = self.isso.conf.get("admin", "password")
+
+        return username == 'admin' and password == admin_password
+
+
+    """
+    @api {get} /pending pending
+    @apiGroup Comment
+    @apiName pending
+    @apiVersion 0.13.0
+    @apiDescription
+        Get the latest comments from the system waiting moderation, no matter which thread. Only available if `[general] pending-enabled` is set to `true` in server config.
+
+    @apiQuery {Number} limit
+        The quantity of last comments to retrieve
+
+    @apiExample {curl} Get the latest 5 pending comments
+        curl 'https://comments.example.com/pending?limit=5'
+
+    @apiUse commentResponse
+
+    @apiSuccessExample Example result:
+        [
+            {
+                "website": null,
+                "uri": "/some",
+                "author": null,
+                "parent": null,
+                "created": 1464912312.123416,
+                "text": " &lt;p&gt;I want to use MySQL&lt;/p&gt;",
+                "dislikes": 0,
+                "modified": null,
+                "mode": 2,
+                "id": 3,
+                "likes": 1
+            },
+            {
+                "website": null,
+                "uri": "/other",
+                "author": null,
+                "parent": null,
+                "created": 1464914341.312426,
+                "text": " &lt;p&gt;I want to use MySQL&lt;/p&gt;",
+                "dislikes": 0,
+                "modified": null,
+                "mode": 2,
+                "id": 4,
+                "likes": 0
+            }
+        ]
+    """
+	# If the admin interface is not enabled, people may have not changed
+	# the default password.  We therefore disallow the /pending endpoint,
+    # as well.
+    @requires_admin
+    @requires_auth
+    def pending(self, environ, request):
+        # if the feature is not allowed, don't present the endpoint
+        if not self.conf.getboolean("pending-enabled"):
+            return NotFound(
+                "Unavailable because 'pending-enabled' not set by site admin"
+            )
+
+        return self._latest(environ, request, "2")
```

Comment:
Rather than adding a new endpoint, wouldn't it make more sense to instead pass through a `mode` GET parameter? It would remain at `"1"` as a default for `/latest` and could be set to `"2"` to get pending comments (which would then require auth).